modules/Bio/EnsEMBL/Compara/RunnableDB/EpoLowCoverage/ImportAlignment.pm

```diff
@@ -158,9 +158,9 @@ sub importAlignment {
                     "ORDER BY node_id DESC";
     }
 
-    my $gat_rows = copy_data($self->param('from_dbc'), $self->compara_dba->dbc,
+    copy_data($self->param('from_dbc'), $self->compara_dba->dbc,
               "genomic_align_tree",
-              $gat_sql, 1, 0, 'ignore_fks', $self->debug);
+              $gat_sql, 1, 'skip_disable_vars', $self->debug);
```

Comment:
I think there is a problem here. This is the `copy_data` call that was using `copy_data_pp` + `SET FOREIGN_KEY_CHECKS = 0` (`ignore_fks` was set) because the internal foreign keys of the `genomic_align_tree` table are otherwise a nightmare to sort out. With `skip_disable_vars` set, the new `copy_data` *won't* do `SET FOREIGN_KEY_CHECKS = 0`, so I think it shouldn't be set ?
```suggestion
              $gat_sql, 1, 0, $self->debug);
```

Reply:
Why did you dismiss this change ?

Reply:
Because I misread the meaning of `'ignore_fks'`.